src/Microsoft.AspNetCore.SignalR.Core/Proxies.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
-    public class UserProxy<THub> : IClientProxy where THub : Hub
+    internal class UserProxy<THub> : IClientProxy where THub : Hub
```

Comment:
Should these be moved to an internal namespace?

Reply:
Yeah, you're right